api/tests/test_api.py

```diff
@@ -225,19 +225,20 @@ def test_bulk_inspect_name_post(test_client):
                 'beautiful_name',
                 'subtitle',
                 'title',
+                'inspected',
             }
             for r in res_json['results']
         ]
     )
 
-    # more than 250 names
+    # more than MAX_NUMBER_OF_NAMES_IN_BULK names
     response = test_client.post('/bulk-inspect-names', json={'names': names * 126})
```

Comment:
Is this really more than `MAX_NUMBER_OF_NAMES_IN_BULK`? I see the number `126` here.

Reply:
Yes, `len(names)==2` so it is 252. I have added assert.